qiskit/ignis/experiments/calibration/cal_base_experiment.py

```diff
@@ -33,18 +33,14 @@ def schedules(self, backend: BaseBackend) -> List[pulse.Schedule]:
         # TODO: Add transpiler & scheduler options
 
         # Get schedule
-        circuits = transpile(self.generator.circuits(),
-                             backend=backend,
-                             initial_layout=self.generator.qubits)
+        circuits = transpile(self.generator.circuits(), backend=backend)
```

Comment:
This is because now InstDef returns transpiled circuit? (number of qubit equals to the device qubit)

Reply:
Yes.